cluster/gce/gci/configure-helper.sh

```diff
@@ -852,6 +852,12 @@ function assemble-docker-flags {
     docker_opts+=" --live-restore=false"
   fi
 
+  # Override docker storage driver if the environment variable is set
+
+  if [[ -n "${DOCKER_STORAGE_DRIVER:-}" ]]; then
+    docker_opts+=" --storage-driver=${DOCKER_STORAGE_DRIVER}"
```

Comment:
By the time we reconfigure we have write the docker options we have already loaded the docker images for some of the kube-components:

https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/gci/master.yaml#L31

Changing the storage driver requires reloading of the images. This change breaks GKE (although it's not the only thing currently breaking GKE). I'd like to revert it in https://github.com/kubernetes/kubernetes/pull/55624 so we can start getting signal in GKE again. cc @abgworrall

Reply:
Ok, we decided to revert this pr.